akka-actor/src/main/scala/akka/event/Logging.scala

```diff
@@ -864,11 +866,8 @@ object Logging {
   trait StdOutLogger {
 
     import StdOutLogger._
-    import java.text.SimpleDateFormat
-    import java.util.Date
 
-    private val date = new Date()
-    private val dateFormat = new SimpleDateFormat("MM/dd/yyyy HH:mm:ss.SSS")
+    private val formatter = DateTimeFormatter.ofPattern("MM/dd/yyyy HH:mm:ss.SSS")
```

Comment:
This is unfortunately not binary compatible because the way traits are compiled, but it's easy to fix. Move the `formatter` up above `trait StdOutLogger`.